pysb/pathfinder.py

```diff
@@ -2,6 +2,9 @@
 import sys
 import sysconfig
 
+# Set to False to not utilize the system PATH environment variable
+use_path = True
```

Comment:
This doesn't feel like the right interface. What sort of situations would you expect people would want to turn this off? I think it would be user- or system-specific, not model- or script-specific. That would point to using an environment variable to control this rather than a Python global. Actually, perhaps both would be useful:
```python
use_path = 'PYSB_PATHFINDER_IGNORE_PATH' not in os.environ
```
This would initialize to True if the user didn't have that environment variable set, otherwise False. The user's Python code could then override it later.

Reply:
Sounds good. Implemented as suggested.